net/libsrtp/Portfile

```diff
@@ -4,20 +4,20 @@ PortSystem          1.0
 PortGroup           muniversal 1.0
 
 name                libsrtp
-version             2.1.0
-revision            2
+version             2.4.2
+revision            0
 categories          net devel
 maintainers         {db.org:aeh @alfredh} openmaintainer
 description         Secure Real-time Transport Protocol (SRTP)
 long_description    The libSRTP library is an open-source implementation of \
                     the Secure Real-time Transport Protocol (SRTP).
 homepage            https://github.com/cisco/libsrtp
-platforms           darwin
 master_sites        https://github.com/cisco/libsrtp/archive/
 license             BSD
 
-checksums           rmd160  436b1167489562ac7e3360136182ce4e28078ceb \
-                    sha256  0302442ed97d34a77abf84617b657e77674bdd8e789d649f1cac0c5f0d0cf5ee
+checksums           rmd160  377d8559d2d3647485d256de44336fe21dc18821 \
+                    sha256  3b1bcb14ebda572b04b9bdf07574a449c84cb924905414e4d94e62837d22b628 \
+                    size    631393
 
 depends_build-append  port:pkgconfig
 depends_lib-append                 \
```

Comment:
instead of the `path:lib/libssl.dylib:openssl` you can/should probably import the `openssl` PG at the top of the Portfile and only set the `openssl.branch` in the `libsrtp1` subport